content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,38 @@ product form.
 
 .. _sales/advanced-price-rules:
 
+Discounts
+=========
+
+The :guilabel:`Discounts` feature allows the ability to set a discount or increase the price
+on *individual items* in a sales order. This is calculated as a percentage of that product's sales
+price.
+
+To access discounts, navigate to the *Sales* app setting page
+(:menuselection:`Sales app --> Configuration --> Settings`), and click the :guilabel:`Discounts`
+checkbox, then click :guilabel:`Save`.
+
+.. image:: pricing/discounts-option.png
+   :align: center
+   :alt: Click the checkbox labeled `Discounts` to activate Discounts.
+
+To set a discount after discounts have been activated, navigate to the list of sales orders by
+going to :menuselection:`Orders --> Orders` at the top of the page. Once there, click on the
+desired sales order from the list.
+
+In the :guilabel:`Order Lines` section of the sales order form, a new heading will appear labeled
+:guilabel:`Disc.%`. This column is used to set discounts on individual items. Enter the desired
+discount for each product line and the new price will automatically be calculated in the sales order
+total at the bottom of the page.
```

Comment:
While discounts can indeed be added to sales orders before the customer is invoiced, they're usually added to quotations first as a means to incentivize a deal to close faster.

Consider the following changes to capture the customer's use case more accurately:
- switching "sales order" to "quote" or "quotation" in this + following sections
- add a **Tip** admonition block to say discounts can be added to sales orders as well before invoicing a customer.